pandas/core/indexes/multi.py

```diff
@@ -2441,7 +2441,51 @@ def reindex(self, target, method=None, level=None, limit=None, tolerance=None):
 
         return target, indexer
 
-    def get_slice_bound(self, label, side, kind):
+    def get_slice_bound(self, label, side, kind) -> int:
```

Comment:
```suggestion
    def get_slice_bound(self, label, side: str, kind: str) -> int:
```

If you know how to annotate `label` would be great as well